XamlStyler.Mac/CommandHandlers/BatchFormatXamlCommandHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿using MonoDevelop.Components.Commands;
```

Comment:
Can you add our standard header to the top of all code files?

"// © Xavalon. All rights reserved."

Reply:
Added copyright everywhere